index.html

```diff
@@ -175,71 +184,177 @@
 
 ---
 
-## Migrating to Git
+class: middle, center
+
+## When can you start migrating to Git? 
 
-### When can you start? 
 --
-**.okay[Right now!]**
+
+### .okay[Right now!]
+---
+
+## Migrating to Git
+
+### How can you do it?
+
 --
 
-- TFS supports Git and mixing Git/TFVC repos in the same project.
-- Git is the default version control provider for new projects in Azure DevOps (Online & On-Prem).
-- Microsoft advises using Git over TFVC.
+#### 1. Add a Git repository to your TFS Team Project.
+
+- TFS supports mixing Git and TFVC repos in the same project.
 - Integrated Git support in Visual Studio and Visual Studio Code.
   - You will want to learn some basic commands.
 
-- [Comparison of Git and TFVC capabilities](https://docs.microsoft.com/en-us/azure/devops/repos/tfvc/comparison-git-tfvc?view=azure-devops)
+???
+- Git is the default version control provider in TFS and Azure DevOps.
+
+--
+
+#### 2. "Clone" your TFVC into a local Git repository.
+
+- There's a tool for that! .okay[**git-tfs**]
+  - Recommended by [Microsoft](https://docs.microsoft.com/en-us/azure/devops/learn/git/migrate-from-tfvc-to-git#advanced-migrations), [GitLab](https://docs.gitlab.com/ee/user/project/import/tfvc.html) and Git-SCM's [Pro Git Book](https://git-scm.com/book/en/v2/Git-and-Other-Systems-Migrating-to-Git#_git_tfs).
+  - Tool takes a long time to process. (~30 mins for 1 GB TFVC repo)
+
+--
+
+#### 3. Push local Git repo to Git hosting service.
+
+- One basic command: `git push`
 ---
 
 ## Migration Tool
 
-### git-tfs
+### git-tfs - Sample Run
 
-- Recommended by Microsoft and Pro Git Book on [Git-SCM](https://git-scm.com/book/en/v2/Git-and-Other-Systems-Migrating-to-Git#_git_tfs).
+```cs
+# 1. Clone from existing TFS Team Project.
+git tfs quick-clone "http://tfs.intra.dmz:8080/tfs/projectcollection" "$/EWS-SWE" .
 
-- [Microsoft - Migrate from TFVC to Git - Advanced Migrations](https://docs.microsoft.com/en-us/azure/devops/learn/git/migrate-from-tfvc-to-git#advanced-migrations)
-  - [GitHub - git-tfs](https://github.com/git-tfs/git-tfs)
-  - [Using git-tfs to migrate from TFVC to Git](https://git-scm.com/book/en/v2/Git-and-Other-Systems-Migrating-to-Git#_git_tfs)
+# 2. Verify integrity of the cloned files.
+git tfs verify
+
+# 3. Push to remote repository. 
+git remote add origin "http://tfs.intra.dmz:8080/tfs/projectcollection/EWS-SWE/_git/EWS-SWE-Git"
+git push --all origin
+```
```

Comment:
Do you want the list part of the code sample?

Reply:
Which code sample?

Reply:
The `git ....` line 238, 239

Reply:
Ok, then what do you mean by "list part of the code sample"? Those `git remote add` and `git push` commands **are** the code sample that I'm highlighting.

Reply:
If you mean the numbers in the comments, then no. They're just so that people can visually see the "steps" and the commands involved. They could copy-paste this code block into their terminal or in a PowerShell script and it could run.

Reply:
Sure.